lib/improver/tests/nbhood/nbhood/test_BaseNeighbourhoodProcessing.py

```diff
@@ -38,6 +38,8 @@
 from iris.coords import AuxCoord, DimCoord
 from iris.cube import Cube
 from iris.tests import IrisTest
+from iris.coord_systems import (LambertAzimuthalEqualArea,
```

Comment:
I'm getting a pylint error, although travis has ran. Could you check it too.

Reply:
I think this was my fault. Its fine now I've updated my site-init.

Reply:
I don't see this, and neither does @gavinevans.